integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/testsupport/util/LuceneTckBackendFeatures.java

```diff
@@ -58,4 +58,9 @@ public boolean supportsHighlighterUnifiedTypeFragmentSize() {
 	public boolean supportsHighlighterUnifiedPhraseMatching() {
 		return true;
 	}
+
+	@Override
+	public boolean supportsComplementRegexFlag() {
+		return false;
+	}
```

Comment:
I don't think you need to bother with this. Let's remove it for all backends. Elasticsearch will likely remove support when it upgrades to Lucene 10 anyway.

Reply:
mm, ok 😃. I was only wondering if that'll match whatever the "compatible" versions of Elasticsearch/OpenSearch would be at the time we switch to Lucene 10.

Reply:
I meant that we'll remove the feature from all backends, so we'll remove the corresponding tests as well, so we won't need this.

Some Elasticsearch/OpenSearch versions may support the operator, but Hibernate Search won't, so we don't really need to bother with this.

At least that's how I see it.